services/web/server/src/simcore_service_webserver/folders/_groups_handlers.py

```diff
@@ -0,0 +1,162 @@
+""" Handlers for project comments operations
+
+"""
+
+import functools
+import logging
+
+from aiohttp import web
+from models_library.folders import FolderID
+from models_library.users import GroupID, UserID
+from pydantic import BaseModel, Extra, Field
+from servicelib.aiohttp.requests_validation import (
+    parse_request_body_as,
+    parse_request_path_parameters_as,
+)
+from servicelib.aiohttp.typing_extension import Handler
+from servicelib.mimetype_constants import MIMETYPE_APPLICATION_JSON
+from servicelib.request_keys import RQT_USERID_KEY
+
+from .._constants import RQ_PRODUCT_KEY
+from .._meta import api_version_prefix as VTAG
+from ..login.decorators import login_required
+from ..security.decorators import permission_required
+from ..utils_aiohttp import envelope_json_response
+from . import _groups_api
+from ._folders_handlers import FoldersPathParams
+from ._groups_api import FolderGroupGet
+from .errors import FolderAccessForbiddenError, FolderGroupNotFoundError
+
+_logger = logging.getLogger(__name__)
+
+
+class _RequestContext(BaseModel):
+    user_id: UserID = Field(..., alias=RQT_USERID_KEY)  # type: ignore[pydantic-alias]
+    product_name: str = Field(..., alias=RQ_PRODUCT_KEY)  # type: ignore[pydantic-alias]
+
+
+def _handle_folders_groups_exceptions(handler: Handler):
+    @functools.wraps(handler)
+    async def wrapper(request: web.Request) -> web.StreamResponse:
+        try:
+            return await handler(request)
+
+        except FolderGroupNotFoundError as exc:
+            raise web.HTTPNotFound(reason=f"{exc}") from exc
+
+        except FolderAccessForbiddenError as exc:
+            raise web.HTTPForbidden(reason=f"{exc}") from exc
+
+    return wrapper
+
+
+#
+# folders groups COLLECTION -------------------------
+#
+
+routes = web.RouteTableDef()
+
+
+class _FoldersGroupsPathParams(BaseModel):
+    folder_id: FolderID
+    group_id: GroupID
+
+    class Config:
+        extra = Extra.forbid
+
+
+class _FoldersGroupsBodyParams(BaseModel):
+    read: bool
+    write: bool
+    delete: bool
+
+    class Config:
+        extra = Extra.forbid
+
+
+@routes.post(
+    f"/{VTAG}/folders/{{folder_id}}/groups/{{group_id}}", name="create_folder_group"
```

Comment:
these groups are "permission groups" right?

Reply:
yes correct, the same as we have for projects and wallets